nncf/tensorflow/quantization/algorithm.py

```diff
@@ -193,7 +193,8 @@ def build_controller(self, model: tf.keras.Model) -> 'QuantizationController':
         return QuantizationController(model, self.config, self._op_names)
 
     def initialize(self, model: tf.keras.Model) -> None:
-        self._run_range_initialization(model)
+        if self._range_init_params is not None:
+            self._run_range_initialization(model)
```

Comment:
It looks like range initialization is mandatory initialization. Thus, skipping can be lead to a bug in the future.

Reply:
@alexsu52 I wouldn't call it mandatory since the scales could feasible be tuned to appropriate values during training even if default-inited by scale of 1.0

Reply:
@vshampor good point! I've closed the conversation.